testing/btest/Baseline/spicy.spicyz-jit/output

```diff
@@ -1,4 +1,49 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+In file included from /Users/tim/Desktop/projects/zeek/testing/btest/.tmp/test_f8f889238522e372-a3a474a9fddd8515.cc:8:
+/Users/tim/Desktop/projects/zeek/build/src/builtin-plugins/spicy-plugin/include/zeek-spicy/runtime-support.h:712:5: warning: 'bro_int_t' is deprecated: Remove in v6.1. Use zeek_int_t. [-Wdeprecated-declarations]
+    bro_int_t bt = (it >= 0 ? it : std::numeric_limits<::bro_int_t>::max());
```

Comment:
Looks like we need to update the plugin. Either way, this should probably not be part of the baseline?

Reply:
I think the intention was to catch compilation warnings on every platform that zeek runs for CI, but I'd also think those should be caught during the spicy builds instead of the zeek builds.

Reply:
I don't think I can fix this in the context of the btest. The problem is that the output gets reordered depending on each run of the test so the various warnings show up in different places in the file, and the test fails. @bbannier Is there someway I can tell spicy via zeek to not output the compiler output during the run unless it fails?

Reply:
> They are not emitted anymore if one drops e.g., HILTI_DEBUG=compiler

I tried that and it's still emitting the warnings.